frontend/keysets/utils.ts

```diff
@@ -0,0 +1,99 @@
+import { charToCodesMapping } from "../char-codes";
+import { KeySet } from "./keyset";
+
+const simpleEnTextToSpans: Object = {
```

Comment:
`En`?

Reply:
Simple: With basic qwerty layout and no fancy keys
En: English